plugins/aws/workspaces/workSpacesOperationalState.js

```diff
@@ -0,0 +1,60 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'WorkSpaces Healthy Instances',
+    category: 'WorkSpaces',
+    domain: 'Identity and Access Management',
+    severity: 'Medium',
+    description: 'Ensures that the AWS WorkSpaces instances are healthy.',
+    more_info: 'The AWS WorkSpaces service periodically sends status requests to the WorkSpaces instances. An instance is pronounced unhealthy when it fails to respond to these HealthCheck requests.',
+    recommended_action: 'Troubleshoot the workspace when state is unhealthy.',
+    link: 'https://docs.aws.amazon.com/workspaces/latest/adminguide/workspace-maintenance.html',
```

Comment:
the link doesnt look relevant, maybe this can be used 'https://docs.aws.amazon.com/workspaces/latest/adminguide/cloudwatch-dashboard.html'